crates/dojo-core/src/storage/db.cairo

```diff
@@ -69,17 +69,39 @@ mod Database {
 
     fn del(class_hash: starknet::ClassHash, table: u250, query: Query) {
         Index::delete(table, query.into());
+        // TODO: emit delete event
```

Comment:
Good catch. Can we create an issue to track this please?

Reply:
In #329